theme/templates/accounts/sign_up_form.html

```diff
@@ -18,6 +31,10 @@
             {% include "includes/form_field.html" with field=form.username inline_label=True %}
         </div>
         <div class="form-group">
+          <input class="form-control form-required tagsinput" name="organization" {% if profile.organization %}value="{{ profile.organization }}"{% endif %}>
```

Comment:
Is it possible to toggle off the input if there are already two tags? It would need to be toggled back on if the user removes one or both of them.
![image](https://user-images.githubusercontent.com/2448568/32675889-c1d6b114-c615-11e7-9a85-19a3e4c2d12c.png)

Reply:
Fixed by simply removing the 2 org limit